src/ip_notifier.cpp

```diff
@@ -63,21 +107,21 @@ namespace libtorrent { namespace aux
 #endif
 	}
 
-	ip_change_notifier::~ip_change_notifier()
+	ip_change_notifier_impl::~ip_change_notifier_impl()
 	{
 #if defined TORRENT_WINDOWS && !defined TORRENT_BUILD_SIMULATOR
 		cancel();
 		m_hnd.close();
```

Comment:
it seems a bit redundant to call ``close`` immediately before destructing the handle